src/ocean/io/model/SuspendableThrottlerCount.d

```diff
@@ -33,6 +33,7 @@ module ocean.io.model.SuspendableThrottlerCount;
 *******************************************************************************/
 
 import ocean.transition;
+import ocean.core.Verify;
```

Comment:
>they don't protect from any kind of memory corruption

But they are valid sanity checks, which is the purpose of `verify`. I don't see any harm leaving them as `verify`.

Reply:
This is what I think as well. Perhaps the trick is "hot path"?

Reply:
No, purpose of `verify` is specifically sanity checking you _can not abandon_. So that when you build `-release` you get maximum performance possible while still not compromising the app. No more, no less.

Reply:
In other words, "sanity check that is nice to have to catch weird app state but which won't help to keep the app working and won't prevent anything bad" is pretty much definition of `assert` use case.

Reply:
Huh... this makes it a very tricky thing to decide, in many cases. A bug in this class, for example, could lead to seriously wrong app behaviour. Can those checks be abandoned?

Also: I was viewing this conversion process as allowing us to run release builds of D2 apps without losing any of the sanity checks that are in place right now but also without the risk of `Error`s being thrown. This is a somewhat different (and I would argue far more pressing) goal.